### ISSUE TITLE
Some question of InnerProductDecoder?

### ISSUE THREAD

**Issue by wzc118** [zfjsail/gae-pytorch#1]
`self.dc = InnerProductDecoder(dropout, act=lambda x: x)`
Why not use `act=torch.sigmoid`here?

**Comment by zfjsail** [comment]
sigmoid has been incorporated into the loss function
`    cost = norm * F.binary_cross_entropy_with_logits(preds, labels, pos_weight=pos_weight)
`
